librz/core/vmlinux.c

```diff
@@ -0,0 +1,164 @@
+#include <rz_core.h>
+#include <rz_util/rz_str.h>
+#include <rz_util/rz_assert.h>
+#include "vmlinux.h"
+#include <stdio.h>
+
+static void add_config(RzVmlinuxConfigTable* config_tbl, char* config_name, char* config_value);
+
+RZ_API bool vmlinux_parse_apply_config_file(const char* config_filepath, RzVmlinuxConfigTable* config_tbl) {
+    rz_return_val_if_fail(config_filepath && config_tbl, false);
```

Comment:
If you add assert, please also add attributes like `RZ_NONNULL`

Reply:
also add `rz_` prefix for any new functionality that is `RZ_API`

Reply:
added both prefix and NONNULL